UNRELEASED.md

```diff
@@ -15,6 +15,7 @@ Use [the changelog guidelines](https://git.io/polaris-changelog-guidelines) to f
 
 ### Bug fixes
 
+- Fix bug in Safari where button text is gray instead of white for enabled `Button` ([#4270](https://github.com/Shopify/polaris-react/pull/4270))
```

Comment:
Can we elaborate a bit here? Maybe mention that it happens during a state change from disabled to enable and vice versa